integrations/deploy.yaml

```diff
@@ -0,0 +1,48 @@
+- &linux
+  id: deploy-linux-generic
+  meta: &linux_meta
+    name: Linux
+    link: ''
+    categories:
+      - deploy.operating-systems
+    icon_filename: ''
+  keywords:
+    - linux
+  description: 'Run the following command on your node to install and claim Netdata:"r'
+  methods:
+    - method: wget
```

Comment:
I cannot see `wget2` in the list, should we also start recommend it?

Reply:
@thiagoftsm: We don't yet use wget2 for kickstart install and has never been positioned on the Cloud. Do we have this supported and is it backwards compatible with wget?

Reply:
Hello @sashwathn ,

Yes, it its compatible and the command line to save our kickstart locally is more or less the same:

```sh
$ wget2 -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh
```

I believe I already talked about it with our @netdata/agent-sre when it was released.

This is not a blocker, it was more a check.

Best regards!

Reply:
I strongly suspect that users who are conciously choosing to use `wget2` will be smart enough to handle this.

That said, we _may_ want to add support to the kickstart script itself for this, though that should a completely separate PR.